src/client.zig

```diff
@@ -65,7 +65,8 @@ fn usage(args: *c.fuse_args) void {
 
 pub fn main(argv: [][*:0]u8, is_test: bool) !void {
     defer log(.info, "client exit\n", .{});
-    const allocator = if (is_test) std.testing.allocator else std.heap.c_allocator;
```

Comment:
We could use `builtin.is_test` instead. This will let us eliminate `is_test` function parameter both in `client.main` and `server.main`.